pkg/scaffold/resource.go

```diff
@@ -145,3 +150,8 @@ func (r *Resource) checkAndSetVersion() error {
 	}
 	return nil
 }
+
+func (r *Resource) setImportGroup() {
+	s := strings.ToLower(r.Group)
+	r.GoImportGroup = strings.Replace(s, "-", "", -1)
```

Comment:
Nit: Rather than doing this initialization in a separate function, I think we should include these lines in `checkAndSetGroups()` around line 134 so that we have all of the group-related validation and initialization together.